pkg/util/util.go

```diff
@@ -522,6 +525,32 @@ func GetVMConfig(env *hfv1.Environment, vmt *hfv1.VirtualMachineTemplate) map[st
 	return config
 }
 
+func GetK8sDnsServer() string {
+	provisionDNS := "10.43.0.10"
+	nameserver := os.Getenv("HF_K8S_NAMESERVER")
+	if nameserver != "" {
+		provisionDNS = nameserver
+	}
+	return provisionDNS
+}
```

Comment:
We should not be hard coding kubernetes dns - or _any_ dns - in here. Our code should not have to concern itself with dns, and also k8s dns is not guaranteed to exist on that IP.